math/vecops/inc/ROOT/RVec.hxx

```diff
@@ -741,6 +838,15 @@ RVec<typename RVec<T>::size_type> Argsort(const RVec<T> &v)
 }
 
 /// Return elements of a vector at given indices
+///
+/// Example code, at the ROOT prompt:
+/// ~~~{.cpp}
+/// using namespace ROOT::VecOps;
+/// RVec<double> v {2., 3., 1.};
+/// auto vTaken = Take(v, {0,2});
+/// vTaken
+/// // (ROOT::VecOps::RVec<double>) { 2.0000000, 1.0000000 }
```

Comment:
```suggestion
/// // (ROOT::VecOps::RVec<double>) { 1.0000000, 3.0000000 }
```

Reply:
Probably lower the precision for a nicer output in the doxygen?

Reply:
I would not. People often complain about what they get at the prompt and what we say they'd get. So, probably lower the precision in the value printer? ;-)